README.md

```diff
@@ -24,17 +24,17 @@
 
 ### Installation
 ```python
-pip install devolearn
+pip install git+https://github.com/DevoLearn/devolearn
```

Comment:
it is known that the current version of devolearn on PyPI is broken, but this installation is should not be the official way to do it for the end users. Can you just add this command below the original  `pip install devolearn` and mention something like "installing the latest version from source" (because that is exactly what this command does)

I will also update devolearn on PyPI soon ^^''

Reply:
Agree :)

Reply:
thank you for the opinion